SCAutolib/models/user.py

```diff
@@ -110,13 +117,31 @@ def cnf(self):
         self._cnf = None
 
     def delete_user(self):
-        logger.info(f"Deleting the user {self.username}")
-        run(['userdel', '-f', self.username], check=True)
-
-    def add_user(self):
-        run(['useradd', '-m', '-p', self.password, self.username], check=True)
-        self.card_dir = f"/home/{self.username}"
-        logger.info(f"Creating new user {self.username}")
+        try:
+            pwd.getpwnam(self.username)
+            logger.info(f"Deleting the user {self.username}")
+            run(['userdel', '-f', self.username], check=True)
+        except KeyError:
+            pass
+        logger.info(f"User {self.username} does not present on the system")
+
+    def add_user(self, force=False):
```

Comment:
If we are adding some logic in creating and deleting users, we may want to add also some unit-tests.

Reply:
Yes. But again, the same resolution as for [this](https://github.com/x00Pavel/SCAutolib/pull/76/files/20aee890b823321df2574a95fedde6551ae6d31d#r908237558) comment, this PR is about the controller and I'm trying to not make it anormous